guides/advanced_guides/configuration.md

```diff
@@ -232,3 +236,28 @@ Depending on the OS, it is either the size that will be allocated on launch or t
 [To know more about storage in MeiliSearch look at this guide](/resources/about_storage.md)
 
 **Default value**: `107374182400` (100 GiB)
+
+### Dumps folder
+
+**Environment variable**: `MEILI_DUMPS_FOLDER`
+**CLI option**: `--dumps-folder`
+
+Path of the folder where dumps will be created if the [dump route](/references/dump.md#trigger-dump) is called.
+
+**Default value**: `dumps/`
+
+### Import dump
+
+**Environment variable**: `MEILI_IMPORT_DUMP`
+**CLI option**: `--import-dump`
+
+Import a dump from the specified path, must be a `.tar.gz` file.
+
+### Dump batch size
+
+**Environment variable**: `MEILI_DUMP_BATCH_SIZE`
+**CLI option**: `--dump-batch-size`
+
+The batch size used in the importation process, the bigger it is the faster the dump is created.
```

Comment:
Document indexation during dump importation are batched:
this mean that if you have `--dump-batch-size 1000`  and the dump contains 2600 documents, the engine will
1) index documents 0 -> 999 (1000 docs)
2) index documents 1000 -> 1999 (1000 docs)
3) index documents 2000 -> 2599 (600 docs)

instead of indexing all of 3600 docs in same time.

more you index documents in the same batch less time it takes but more memory is used.

Reply:
I'm sorry I was not explicit. Could you add that to the documentation?

```suggestion
The batch size used in the importation process. The more you index documents in the same batch the less time it takes but more memory is used.

**Example**
Imagine you have `--dump-batch-size 1000` and the dump contains 2600 documents. Instead of indexing all of 3600 docs in one go, the engine will :

1. Index documents 0 -> 999 (1000 docs)
2. Index documents 1000 -> 1999 (1000 docs)
3. Index documents 2000 -> 2599 (600 docs)
```